src/_guides/language/language-tour.md

```diff
@@ -720,6 +721,83 @@ information about lists, see [Generics](#generics) and
 [Collections](/guides/libraries/library-tour#collections).
 
 
+### Sets
+
+A set in Dart is an unordered collection of unique items.
+Dart support for sets is provided by set literals and the [Set][] type.
+
+<aside class="alert alert-info" markdown="1">
+  **Version note:** Although the Set _type_ has always been a core part of Dart,
+  set _literals_ were introduced in Dart 2.2.
+</aside>
+
+Here is a simple Dart set, created using a set literal:
+
+<?code-excerpt "misc/lib/language_tour/built_in_types.dart (set-literal)"?>
+{% prettify dart %}
+var halogens = {'fluorine', 'chlorine', 'bromine', 'iodine', 'astatine'};
+{% endprettify %}
+
+<aside class="alert alert-info" markdown="1">
+  **Note:**
+  The analyzer infers that `halogens` has the type
+  `Set<String>`. If you try to add the wrong type of value
+  to the set, the analyzer or runtime raises an error.
+  For more information, read about
+  [type inference.](/guides/language/sound-dart#type-inference)
+</aside>
+
+To create an empty set, use `{}` preceded by a generic type annotation,
```

Comment:
"a generic type annotation" -> "a type argument"
(if you mean to write, e.g., `<int>{}`, but maybe just write that example instead of only describing it in prose).

Ob-pedantry: Something is generic if it works for multiple types, usually meaning that it is parameterized by type. The `List` class is generic, but `List<int>` is not. A "generic type argument" is not really correct, but semantically close enough that the meaning is clear - a type argument for something generic, even if the type argument is not *itself* generic).
Also, `<int>` is a type argument, not a type annotation. Type annotations are types like `int x;` in a variable declaration.

Reply:
Done